projectRouter.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+
+const projectRouter = express.Router();
+
+const projectDB = require('./data/helpers/projectModel');
+
+projectRouter.get('/', (req,res) => {
+    projectDB.get()
+    .then( projects => {
+        res.status(200).json(projects);
+    })
+    .catch( err => {
+        res.status(500).json(err);
+    })
+})
+
+projectRouter.get('/:id', (req,res) => {
+    const id = req.params.id;
+    projectDB.get(id)
+    .then( project => {
+        if (project) {
+            res.status(200).json(project)
+        } else {
+            res.status(404).json({error : `project with id # ${id} was not found`})
+        }
+    })
+    .catch( err => {
+        res.status(500).json(err.message);
+    })
+})
+
+projectRouter.post('/', (req,res) => {
+    const newProject = req.body;
+    projectDB.insert(newProject)
+    .then( newlyCreatedProject => {
+        res.status(201).json(newlyCreatedProject);
+    })
+    .catch( err => {
+        res.status(500).json(err.message);
+    })
+})
+
+projectRouter.put('/', (req,res) => {
+    const id = req.body.id;
```

Comment:
as you have been using the url params in the majority of the endpoints here you will want to be consistent. pick one or the other, but if you are trying to architech a more RESTful api, you will want to use the url params and give unique endpoints for every resource, not use the same endpoint with different data being passed each request

Reply:
Yea that makes sense.  Just getting a hang for how it all works, as i get more comfortable I'll get more consistent with the way I design things.